cpp/tests/Graph/Graph.cu

```diff
@@ -102,6 +102,7 @@ TEST(gdf_edge_list, success_no_weights)
 }
 */
 
+#if 0
```

Comment:
We should have some equivalent stand-alone tests for testing the new graph structure. This can be in another PR though. 
In the meantime, I think it is ok to delete dead code. We can still access it from Github history.

Reply:
Removed this code.